packages/foreman/foreman/foreman.spec

```diff
@@ -711,6 +713,34 @@ Meta Package to install requirements for Redis caching support
 %files redis
 %{_datadir}/%{name}/bundler.d/redis.rb
 
+%package dynflow-sidekiq
+Summary: Foreman Redis caching support
+Group:  Applications/System
+# start specfile redis Requires
+Requires: %{?scl_prefix}rubygem(sidekiq) >= 5.0
+Requires: %{?scl_prefix}rubygem(sidekiq) < 6
+Requires: %{?scl_prefix}rubygem(gitlab-sidekiq-fetcher)
+# end specfile redis Requires
+Requires: %{name} = %{version}-%{release}
+
+%description dynflow-sidekiq
+Meta Package to install requirements for Redis caching support
```

Comment:
This description mentions Redis caching support but this is for Dynflow support right?

Reply:
My bad, too much of copy pasting.